### PR TITLE
feat(project): Project Service (DSP-1478)

### PR DESCRIPTION
**Pull request by mdelez**
resolves DSP-1478

### CODE REVIEW

**Comment by mdelez** [comment]
first try! 🥳

**Comment by mdelez** [comment]
@subotic @BalduinLandolt There's a lot to review here and definitely many things that can be improved but I think it is a good starting point. Should we make a meeting to review this all together?

**Review by subotic — APPROVED** [review]

File: services/admin/backend/api/presenter/project.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *  it will be useful, but WITHOUT ANY WARRANTY; without even
+ *  the implied warranty of MERCHANTABILITY or FITNESS FOR
+ *  A PARTICULAR PURPOSE.  See the GNU Affero General Public
+ *  License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public
+ *  License along with the DaSCH Service Platform.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
```
Comment (BalduinLandolt) [inline services/admin/backend/api/presenter/project.go:22]: wrong license :)

File: services/admin/backend/entity/project/error.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *  it will be useful, but WITHOUT ANY WARRANTY; without even
+ *  the implied warranty of MERCHANTABILITY or FITNESS FOR
+ *  A PARTICULAR PURPOSE.  See the GNU Affero General Public
+ *  License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public
+ *  License along with the DaSCH Service Platform.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
```
Comment (BalduinLandolt) [inline services/admin/backend/entity/project/error.go:22]: wrong license

**Review by BalduinLandolt — APPROVED** [review]
Please change the remaining A-GPL licenses to Apache... But apart from that, looks good, thanks!

**Pull request merged**